ibis/backends/polars/compiler.py

```diff
@@ -1006,70 +1006,70 @@ def bitwise_not(op):
 
 
 @translate.register(ops.Binary)
-def binop(op):
-    left = translate(op.left)
-    right = translate(op.right)
+def binop(op, **kw):
+    left = translate(op.left, **kw)
+    right = translate(op.right, **kw)
     func = _binops.get(type(op))
     if func is None:
         raise com.OperationNotDefinedError(f'{type(op).__name__} not supported')
     return func(left, right)
 
 
 @translate.register(ops.ElementWiseVectorizedUDF)
-def elementwise_udf(op):
-    func_args = list(map(translate, op.func_args))
+def elementwise_udf(op, **kw):
+    func_args = list(map(partial(translate, **kw), op.func_args))
```

Comment:
Can you expand it into a list comprehension?

Reply:
Sure!

Reply:
Done.